sdk/webpubsub/azure-messaging-webpubsubservice/CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Release History
 
+## 1.3.0 (Unreleased)
+
+### Features Added
+- Added new APIs "list_connections_in_group"
```

Comment:
I didn't see in the PR where these APIs were added. Was that done in a different PR?

Reply:
Hi, the method is generated in the "_operations.py" file for both sync and async modes. And it's customized in the "_patchs.py" files.